tests/components/MegadraftEditor_test.js

```diff
@@ -624,7 +623,6 @@ describe("MegadraftEditor Component", () => {
     );
     const toolbar = wrapper.find(MyCustomToolbar);
     expect(toolbar).to.have.length(1);
-    expect(toolbar.prop("actions")).to.equal(this.component.actions);
```

Comment:
The same here, the toolbar still needs to receive `actions` so the `expect` is still needed.